AuroraEditor/Base/AppPreferences/Controller/PreferencesWindowController.swift

```diff
@@ -16,7 +16,6 @@ final class PreferencesWindowController: NSWindowController, NSToolbarDelegate {
         self.init(window: window)
         window.title = "Preferences"
         window.styleMask.insert(.fullSizeContentView)
-        window.styleMask.remove(.resizable)
```

Comment:
I am not sure if static window size is valuable or not. To solve #350 , I made this line deleted.